src/pages/Portfolio/index.js

```diff
@@ -337,7 +337,6 @@ const Portfolio = () => {
               get(userStats, `[${stakedVaults[i]}]['lpTokenBalance']`, 0),
               (fAssetPool && fAssetPool.lpTokenData && fAssetPool.lpTokenData.decimals) || 18,
               POOL_BALANCES_DECIMALS,
-              true,
             )
```

Comment:
It should include true, not sure for me 
@devstar0718 
Could you check it?

Reply:
I see.
If that value is true, it is using 'formatNumber'.
Also I used 'formatNumber' function in render section.
So you mean I used this function double times. right?

But it's not big problem. It has same result although that field is true or false.

Also I am checking myself about my work.
I will fix them if there are serious bugs.

I will be responsible for my work.

Thanks for your checking.

Reply:
I made this change. There's a problem when formatting twice for numbers larger than 1000. Because these get changed to 1.000k by formatting, and the second time they don't get recognised as a number so it displays 0.

So with this fix it shows all numbers correctly on the portfolio page